Copyedit sumarizacao narrativas web python

joanacvp opened the pull request.

joanacvp committed:
Update sumarizacao-narrativas-web-python.md

Revisão geral feita. Irei realizar ainda mais uma leitura final

joanacvp committed:
Update sumarizacao-narrativas-web-python.md

Finished the copy editing.

joanacvp commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
+# *Conta-me Histórias*
 
-O projeto Conta-me Histórias é um projeto desenvolvido por investigadores do Laboratório de Inteligência Artificial e Apoio a Decisão ([LIAAD](https://www.inesctec.pt/en/centres/liaad) — [INESCTEC](https://www.inesctec.pt/en/)) e afiliados às instituições [Instituto Politécnico de Tomar](https://www.ipt.pt) — [Centro de Investigação em Cidades Inteligentes (CI2)](http://www.ci2.ipt.pt/pt/); [Univ ...
+O projeto *Conta-me Histórias* é desenvolvido por pesquisadores do Laboratório de Inteligência Artificial e Apoio a Decisão ([LIAAD](https://www.inesctec.pt/pt/centros/liaad)) — [INESCTEC](https://www.inesctec.pt/pt)) e afiliados às instituições [Instituto Politécnico de Tomar](https://www.ipt.pt) — [Centro de Investigação em Cidades Inteligentes (CI2)](http://www.ci2.ipt.pt/pt/), [Universidade d ...
```
O link no final da frase não está a abrir (https://www.youtube.com/embed/fcPOsBCwyu8).

anisa-hawes replied:
Hello @joanacvp,

In this case, I think the correct link would be: https://youtu.be/fcPOsBCwyu8

joanacvp replied:
Thank you very much @anisa-hawes. I already changed it :)

joanacvp commented:
Boa tarde @dcgomes e @josircg ,

Realizei o codyedit desta lição. Poderão verificá-lo no separador "Files changes" acima.
Chamo a atenção que no final da linha 258 o link não está a abrir (https://www.youtube.com/embed/fcPOsBCwyu8) - deixei comentário nessa parte.

Fico a aguardar o vosso feedback. Estou disponível para qualquer esclarecimento.

Muito obrigada,
Joana

joanacvp committed:
Update sumarizacao-narrativas-web-python.md

Changed the link on line 258

anisa-hawes commented:
Olá @dcgomes e @josircg,

Muito obrigada por fornecer textos alt-texto descritivos para acompanhar as imagens das figuras!

Existem alguns recursos portugueses que eu possa acrescentar ao nosso Wiki como orientação para outros?

Obrigada,
Anisa

josircg commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -27,94 +27,79 @@ doi: A INDICAR
 # Pré-requisitos
 
-A participação neste tutorial pressupõe conhecimentos básicos ao nível da programação (nomeadamente Python) bem como familiarização com a instalação de packages python (via [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)), [formato JSON](https://www.w3schools.com/js/js_json_intro.asp) e consumo de APIs. A execução do código pressupõe o recurso ao jupyter anaconda. Para uma in ...
+O desenvolvimento deste tutorial pressupõe conhecimentos básicos de programação (nomeadamente Python) bem como familiarização com a instalação de pacotes python (via [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) (em inglês)), com o [formato JSON](https://www.w3schools.com/js/js_json_intro.asp) (em inglês) e com o consumo de APIs. A execução do código pressupõe o recurso ao  ...
```
Acho que "Participação" cabe melhor que o desenvolvimento. Quem "desenvolveu" foram os autores.

josircg replied:
O correto seria Jupyter Notebook. O Anaconda é um dos softwares que auxiliam na instalação do Jupyter.

joanacvp replied:
Muito obrigada @josircg. Alterações introduzidas.

josircg commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -27,94 +27,79 @@ doi: A INDICAR
-O Arquivo.pt foca-se na preservação de informação publicada na web de interesse para a comunidade portuguesa. Porém, contém páginas escritas em várias línguas de interesse para a comunidade internacional e cerca de metade dos seus utilizadores são oriundos de fora de Portugal. 
+O [Arquivo.pt](https://www.arquivo.pt) é um serviço público e gratuito disponibilizado pela [Fundação para a Ciência e a Tecnologia I.P.](https://www.fct.pt), que permite a qualquer pessoa pesquisar e aceder a informação histórica preservada da web desde os anos 90. Embora se foque na preservação de informação de interesse para a comunidade portuguesa, contém também páginas escritas em várias lín ...
 
 [Este vídeo](https://www.youtube.com/embed/EnSys0HDnCc) introduz brevemente o Arquivo.pt.
```
O link atualizado é esse https://www.youtube.com/watch?v=fcPOsBCwyu8

joanacvp replied:
Muito obrigada @josircg. Link alterado.

josircg commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -436,34 +415,35 @@ for period in summ_result["results"]:
 # Bibliografia
 
-* Campos, R., Pasquali, A., Jatowt, A., Mangaravite, V., Jorge, A. (2021). Automatic Generation of Timelines for Past-Web Events. In Gomes, D., Demidova, E., Winters, J., Risse, T. (Eds.), The Past Web: Exploring Web Archives (pp. 225-242). [pdf](https://link.springer.com/chapter/10.1007/978-3-030-63291-5_18)
+* Campos, R., Pasquali, A., Jatowt, A., Mangaravite, V., and Jorge, A.. "Automatic Generation of Timelines for Past-Web Events" In *The Past Web: Exploring Web Archives*, edited by D. Gomes, E. Demidova, J. Winters, and T. Risse, 225-242. Springer: 2021. [https://link.springer.com/chapter/10.1007/978-3-030-63291-5_18](https://link.springer.com/chapter/10.1007/978-3-030-63291-5_18)
```
Qual é o padrão de bibliografia utilizado pelo ProgHist? APA? Não achei isso em nossa nossa documentação.

anisa-hawes replied:
Hello @josircg. We ask authors to use [Chicago Manual of Style](https://www.chicagomanualofstyle.org/tools_citationguide.html).

joanacvp replied:
Exacto, é o Chicago Manual of Style. Encontra-se referido aqui: [](https://programminghistorian.org/pt/directrizes-autor)

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -27,94 +27,79 @@ doi: A INDICAR
-Na primeira parte do tutorial, iremos apresentar sumariamente as funções de pesquisa e acesso disponibilizadas pelo Arquivo.pt. Demonstraremos como podem ser utilizadas de forma automática através da invocação dos métodos disponibilizados pela Arquivo.pt [API (Application Programming Interface)](https://pt.wikipedia.org/wiki/Interface_de_programa%C3%A7%C3%A3o_de_aplica%C3%A7%C3%B5es) através de e ...
-
-Na segunda parte, recorremos ao Conta-me Histórias para exemplificar o processo de sumarização temporal de um evento. Nesse sentido, demonstraremos a forma como os utilizadores podem obter informações históricas resumidas sobre um determinado tópico ([Jorge Sampaio](https://pt.wikipedia.org/wiki/Jorge_Sampaio), presidente da república de Portugal entre 1996 e 2006) que envolva notícias do passado ...
+Na segunda parte, recorremos ao *Conta-me Histórias* para exemplificar o processo de sumarização temporal de um evento. Nesse sentido, demonstraremos a forma como os usuários podem obter informações históricas resumidas sobre um determinado tópico (por exemplo, sobre [Jorge Sampaio](https://pt.wikipedia.org/wiki/Jorge_Sampaio), presidente da República Portuguesa entre 1996 e 2006), que envolva no ...
```
Não gosto muito da palavra "usuário". Prefiro "utilizador"

joanacvp replied:
Bom dia @rncampos. Tendo em conta os públicos PT e BR, foi feito um balanço dos vários termos e conceitos por ambas as partes. Tanto no caso do "utilizador", como de "investigação" e "investigador" optou-se pelos termos BR (usuário, pesquisa e pesquisador) de forma a haver homogeneidade nas lições. Por esse motivo foram realizadas as respetivas alterações, segundo essas mesmas diretrizes.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -27,94 +27,79 @@ doi: A INDICAR
-- extrair informação relevante a partir do Arquivo.pt fazendo uso da [Arquivo.pt API (Full-text & URL search)](https://github.com/arquivo/pwa-technologies/wiki/Arquivo.pt-API);
-- saber usar o package do Conta-me Histórias (https://github.com/LIAAD/TemporalSummarizationFramework) no contexto da sumarização temporal automática de eventos a partir de elevados volumes de dados preservados no arquivo da web portuguesa.
+- Extrair informação relevante a partir do Arquivo.pt fazendo uso da [Arquivo.pt API (Full-text & URL search)](https://github.com/arquivo/pwa-technologies/wiki/Arquivo.pt-API) (em inglês)
+- Saber usar o pacote do *Conta-me Histórias* (https://github.com/LIAAD/TemporalSummarizationFramework) (em inglês) no contexto da sumarização temporal automática de eventos a partir de elevados volumes de dados preservados no arquivo da web portuguesa
```
"pacote de software Python" ao invés de "pacote". Se bem que preferia: "saber usar a biblioteca Python"

joanacvp replied:
Alterado para "saber usar a biblioteca Python"

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -27,94 +27,79 @@ doi: A INDICAR
+O Arquivo.pt contém milhares de milhões de ficheiros recolhidos ao longo do tempo a partir de websites em várias línguas que documentam eventos nacionais e internacionais. Os serviços de pesquisa que fornece incluem a pesquisa de texto integral, a pesquisa de imagens, a listagem do histórico de versões, a pesquisa avançada e [APIs](https://arquivo.pt/api), que facilitam o desenvolvimento por terc ...
 
-Ao longo dos anos, o Arquivo.pt tem sido utilizado como recurso para suportar trabalhos de investigação em áreas como as Humanidades ou Ciências Sociais. Desde 2018, o [Prémio Arquivo.pt](https://arquivo.pt/premios) distingue anualmente trabalhos inovadores baseados na informação histórica preservada pelo Arquivo.pt. Os investigadores e cidadãos tem vindo a ser sensibilizados para a importância d ...
+Ao longo dos anos, o Arquivo.pt tem sido utilizado como recurso para suportar trabalhos de pesquisa em áreas como as Humanidades ou as Ciências Sociais. Desde 2018, o [Prémio Arquivo.pt](https://arquivo.pt/premios) distingue anualmente trabalhos inovadores baseados na informação histórica preservada pelo Arquivo.pt. Os pesquisadores e cidadãos têm vindo a ser sensibilizados para a importância da  ...
```
Não gosto da palavra "pesquisadores". "investigadores" faz mais sentido

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -27,94 +27,79 @@ doi: A INDICAR
-* [Memento API (URL search): international standard](https://github.com/arquivo/pwa-technologies/wiki/Memento--API)
-
-Para saber detalhes acerca de [todas as APIs disponibilizadas pelo Arquivo.pt](https://arquivo.pt/api) consulte os conteúdos de formação disponíveis em:
+Neste tutorial iremos abordar apenas a utilização da API do Arquivo.pt. Porém, este disponibiliza também outras:
```
Esta frase não me parece estar correta: "Neste tutorial iremos abordar apenas a utilização da API do Arquivo.pt. Porém, este disponibiliza também outras:". Deveria ser: "Neste tutorial iremos abordar apenas a utilização da API Full-text & URL Search do Arquivo.pt. Porém, este disponibiliza também outras APIS:"

joanacvp replied:
Alterado para "Neste tutorial iremos abordar apenas a utilização da API Full-text & URL Search do Arquivo.pt. Porém, este disponibiliza também outras APIs:"

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -157,22 +142,20 @@ for item in contentsJSon["response_items"]:
-No processo de acesso à informação os utilizadores podem definir o intervalo temporal das datas de arquivo das páginas a serem pesquisadas, através da especificação das datas nos parâmetros de pesquisa da API `from` e `to`.
-
-As datas especificadas devem seguir o formato: ano, mês, dia, hora, minuto e segundo (aaaammddhhmmss), por exemplo a data 9 de março de 1996 seria representada por:
+No processo de acesso à informação os usuários podem definir o intervalo temporal das datas de arquivo das páginas a serem pesquisadas, através da especificação das datas pretendidas nos parâmetros de pesquisa da API `from` e `to`. Estas devem seguir o formato: ano, mês, dia, hora, minuto e segundo (aaaammddhhmmss). Por exemplo, a data 9 de março de 1996 seria representada por:
```
uttilizadores ao invés de usuários

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -196,17 +179,15 @@ for item in contentsJSon["response_items"]:
+### Restringir a pesquisa a um determinado website
 
-O seguinte código executa uma pesquisa por "Jorge Sampaio" sobre páginas arquivadas apenas a partir do website com o domínio "www.presidenciarepublica.pt" compreendidos entre março de 1996 e março de 2006, e apresenta os resultados obtidos.
+Se os usuários apenas tiverem interesse na informação histórica publicada por um determinado website, podem restringir a pesquisa através da especificação no parâmetro de pesquisa da API `siteSearch`. O seguinte código executa uma pesquisa por "Jorge Sampaio" de páginas arquivadas apenas a partir do website com o domínio "www.presidenciarepublica.pt", compreendidas entre março de 1996 e março de  ...
```
utilizadores

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -232,24 +213,22 @@ for item in contentsJSon["response_items"]:
+### Restringir a pesquisa a um determinado tipo de ficheiro
 
-Os utilizadores podem definir o tipo do ficheiro sobre o qual a pesquisa deverá incidir através da especificação do parâmetro de pesquisa da API `type`.
+Além de páginas da web, o Arquivo.pt também preserva outros formatos de ficheiro vulgarmente publicados online, como por exemplo documentos do tipo PDF. Os usuários podem definir o tipo de ficheiro sobre o qual a pesquisa deverá incidir através da especificação no parâmetro de pesquisa `type` da API.
```
utilizadores

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -232,24 +213,22 @@ for item in contentsJSon["response_items"]:
+* Entre março de 1996 e março de 2006
 
-E apresenta os resultados obtidos. Quando o utilizador abrir o endereço do conteúdo arquivado fornecido pelo campo de resposta `linkToArchive` terá acesso ao ficheiro PDF.
+E apresenta os resultados obtidos. Quando o usuário abrir o endereço do conteúdo arquivado fornecido pelo campo de resposta `linkToArchive` terá acesso ao ficheiro PDF.
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
+# *Conta-me Histórias*
 
-O projeto Conta-me Histórias é um projeto desenvolvido por investigadores do Laboratório de Inteligência Artificial e Apoio a Decisão ([LIAAD](https://www.inesctec.pt/en/centres/liaad) — [INESCTEC](https://www.inesctec.pt/en/)) e afiliados às instituições [Instituto Politécnico de Tomar](https://www.ipt.pt) — [Centro de Investigação em Cidades Inteligentes (CI2)](http://www.ci2.ipt.pt/pt/); [Univ ...
+O projeto *Conta-me Histórias* é desenvolvido por pesquisadores do Laboratório de Inteligência Artificial e Apoio a Decisão ([LIAAD](https://www.inesctec.pt/pt/centros/liaad)) — [INESCTEC](https://www.inesctec.pt/pt)) e afiliados às instituições [Instituto Politécnico de Tomar](https://www.ipt.pt) — [Centro de Investigação em Cidades Inteligentes (CI2)](http://www.ci2.ipt.pt/pt/), [Universidade d ...
```
investigadores ao invés de pesquisadores
utilizadores ao invés de usuários
Referiu que na linha 258 constava um link do youtube que não abria. Acabei de verificar e a verdade é que ele abre sem qualquer tipo de problema

joanacvp replied:
Respondido acima quando à alteração de "investigadores" e "utilizadores". Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.
Quanto ao link de Youtube já tinha sido alterado anteriormente.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
-- página web (versão EN): http://tellmestories.pt. Ao invés da versão PT que tem por base o Arquivo.pt, a versão EN foi construída tendo por o conjunto de dados designado por [Signal Media Dataset](https://research.signal-ai.com/newsir16/signal-dataset.html) e que reúne mais de 1 milhão de notícias coletadas durante o mês de Setembro de 2015 por empresas de comunicação como a Reuters.
+O projeto *Conta-me Histórias* encontra-se disponível, desde 2018, a partir dos seguintes endereços:
+- Página web (versão PT): https://contamehistorias.pt
+- Página web (versão EN): http://tellmestories.pt (em inglês). Ao contrário da versão PT, que tem por base o Arquivo.pt, a versão EN foi baseada no conjunto de dados designado por [Signal Media Dataset](https://research.signal-ai.com/newsir16/signal-dataset.html) (em inglês), que reúne mais de 1 milhão de notícias coletadas durante o mês de setembro de 2015 por empresas de comunicação como a *Reu ...
```
Retire pf este texto da linha 268 uma vez que esta versão já não funciona

joanacvp replied:
Eliminada a linha 268.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
+- *Conta-me Histórias back-end*: https://github.com/LIAAD/contamehistorias-api (em inglês)
 
-Mais recentemente, em setembro de 2021, o Arquivo.pt passou a disponibilizar a funcionalidade `Narrativa` através da disponibilização de um botão adicional na sua interface que redireciona os utilizadores para o website do Conta-me Histórias para que a partir deste website os utilizadores possam criar automáticamente narrativas temporais sobre qualquer tema. A funcionalidade `Narrativa` é o resul ...
+Mais recentemente, em setembro de 2021, o Arquivo.pt passou a disponibilizar a funcionalidade "Narrativa", através de um botão adicional na sua interface que redireciona os usuários para o website do *Conta-me Histórias*, para que a partir deste possam criar automaticamente narrativas temporais sobre qualquer tema. A funcionalidade "Narrativa" resulta da colaboração entre a equipa do *Conta-me Hi ...
```
utilizadores

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
 ## Como Funciona?
 
-Quando um utilizador insere um conjunto de palavras acerca de um tema na caixa de pesquisa do Arquivo.pt e clica no botão `Narrativa`, é direcionado para o serviço `Conta-me Histórias`, que por sua vez analisa automaticamente as notícias de 26 websites arquivados pelo Arquivo.pt ao longo do tempo e apresenta ao utilizador uma cronologia de notícias relacionadas com o tema pesquisado.
+Quando um usuário insere um conjunto de palavras acerca de um tema na caixa de pesquisa do Arquivo.pt e clica no botão "Narrativa", é direcionado para o serviço *Conta-me Histórias* que, por sua vez, analisa automaticamente as notícias de 26 websites arquivados pelo Arquivo.pt ao longo do tempo e apresenta-lhe uma cronologia de notícias relacionadas com o tema pesquisado.
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
+{% include figure.html filename="sumarizacao-narrativas-web-python-2.jpg" alt="Resultados da pesquisa por Jorge Sampaio no Conta-me Histórias para o periodo compreendido entre 07/04/2016 e 17/11/2016" caption="Figura 2: Resultados da pesquisa por "Jorge Sampaio" no *Conta-me Histórias* para o periodo compreendido entre 2016-04-07 e 2016-11-17." %}
 
-Para a seleção das notícias mais relevantes recorremos ao [YAKE!](http://yake.inesctec.pt), um extrator de palavras relevantes (desenvolvido pela nossa equipa de investigação), e que neste contexto é utilizado para selecionar os excertos mais importantes de uma notícia (concretamente os seus títulos) ao longo do tempo.
+Para a seleção das notícias mais relevantes recorremos ao [YAKE!](http://yake.inesctec.pt) (em inglês), um extrator de palavras relevantes (desenvolvido pela nossa equipa de pesquisa) e que, neste contexto, é utilizado para selecionar os excertos mais importantes de uma notícia (mais concretamente os seus títulos) ao longo do tempo.
```
investigação ao invés de pesquisa

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
+Para a seleção das notícias mais relevantes recorremos ao [YAKE!](http://yake.inesctec.pt) (em inglês), um extrator de palavras relevantes (desenvolvido pela nossa equipa de pesquisa) e que, neste contexto, é utilizado para selecionar os excertos mais importantes de uma notícia (mais concretamente os seus títulos) ao longo do tempo.
 
-Um aspeto interessante da aplicação é o facto de esta facilitar o acesso à página web arquivada que dá nome ao título selecionado como relevante. Por exemplo, ao clicar em cima do título `Jorge Sampaio formaliza apoio a Sampaio da Nóvoa` o utilizador poderá visualizar a seguinte página web:
+Um aspeto interessante da aplicação é o facto desta facilitar o acesso à página web arquivada que dá nome ao título selecionado como relevante. Por exemplo, ao clicar em cima do título "Jorge Sampaio formaliza apoio a Sampaio da Nóvoa" o usuário poderá visualizar a seguinte página web:
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
+{% include figure.html filename="sumarizacao-narrativas-web-python-4.jpg" alt="Nuvem de palavras com os termos relacionados com a pesquisa Jorge Sampaio ao longo de 10 anos" caption="Figura 4: Nuvem de palavras com os termos relacionados com a pesquisa por "Jorge Sampaio" ao longo de 10 anos." %}
 
-O `Conta-me Histórias` pesquisa, analisa e agrega milhares de resultados para gerar cada narrativa acerca de um tema. Recomenda-se a escolha de palavras descritivas sobre temas bem definidos, personalidades ou eventos para obter boas narrativas. No seção seguinte descrevemos a forma como através da biblioteca Python os utilizadores podem interagir e fazer uso dos dados do Conta-me Histórias.
+O *Conta-me Histórias* pesquisa, analisa e agrega milhares de resultados para gerar cada narrativa acerca de um tema. Recomenda-se a escolha de palavras descritivas sobre temas bem definidos, personalidades ou eventos para obter boas narrativas. No seção seguinte descrevemos a forma como, através da biblioteca Python, os usuários podem interagir e fazer uso dos dados do *Conta-me Histórias*.
```
utilizadores

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -332,15 +311,15 @@ Para a instalação da [biblioteca Conta-me Histórias](https://github.com/LIAAD
 ### Definição dos parâmetros de pesquisa
 
-No próximo código o utilizador é convidado a definir o conjunto de parâmetros de pesquisa. A variável `domains` lista o conjunto de 24 websites objeto de pesquisa. Um aspeto interessante desta variável é a possibilidade de o utilizador definir a sua própria lista de fontes notíciosas. Um exercício interessante passa por definir um conjunto de meios de comunicação de âmbito mais regional, por opos ...
+No próximo código o usuário é convidado a definir o conjunto de parâmetros de pesquisa. A variável `domains` lista o conjunto de 24 websites objeto de pesquisa. Um aspeto interessante desta variável é a possibilidade do usuário definir a sua própria lista de fontes noticiosas. Um exercício interessante passa por definir um conjunto de meios de comunicação de âmbito mais regional, por oposição aos ...
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -332,15 +311,15 @@ Para a instalação da [biblioteca Conta-me Histórias](https://github.com/LIAAD
+No próximo código o usuário é convidado a definir o conjunto de parâmetros de pesquisa. A variável `domains` lista o conjunto de 24 websites objeto de pesquisa. Um aspeto interessante desta variável é a possibilidade do usuário definir a sua própria lista de fontes noticiosas. Um exercício interessante passa por definir um conjunto de meios de comunicação de âmbito mais regional, por oposição aos ...
 
-Os parâmetros `from` e `to` permitem estabelecer o espectro temporal de pesquisa. Finalmente na variável `query` o utilizador é convidado a definir o tema de pesquisa (e.g. `Jorge Sampaio`) para o qual pretende construir uma narrativa temporal. Uma vez executado o código o sistema inicia o processo de pesquisa junto do Arquivo.pt. Para tal recorre à utilização da [Arquivo.pt API (Full-text & URL  ...
+Os parâmetros `from` e `to` permitem estabelecer o espectro temporal da pesquisa. Finalmente, na variável `query` o usuário é convidado a definir o tema da pesquisa (e.g., "Jorge Sampaio") para o qual pretende construir uma narrativa temporal. Uma vez executado o código o sistema inicia o processo de pesquisa junto do Arquivo.pt. Para tal, recorre à utilização da [Arquivo.pt API (Full-text & URL  ...
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -351,18 +330,18 @@ apt =  ArquivoPT()
+### Percorrer os resultados obtidos no Arquivo.pt
 
-O objeto `search_result` devolve o número total de resultados obtidos a partir da chamada à API do Arquivo.pt. O número total de resultados excede facilmente as 10,000 entradas. Um tal volume de dados é praticamente impossível de processar por qualquer utilizador que a partir dele queira retirar conhecimento em tempo útil.
+O objeto `search_result` devolve o número total de resultados obtidos a partir da chamada à API do Arquivo.pt. O número total de resultados excede facilmente as 10.000 entradas, um volume de dados praticamente impossível de processar por qualquer usuário que, a partir dele, queira retirar conhecimento em tempo útil.
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -375,9 +354,9 @@ for x in search_result:
+### Determinação de datas importantes e seleção das *keywords*/títulos relevantes
 
-No próximo passo o sistema recorre ao algoritmo do Conta-me Histórias para criar um resumo das notícias mais importantes a partir do conjunto de documentos obtidos do Arquivo.pt. Cada bloco temporal determinado como relevante pelo sistema reúne um total de 20 notícias. Os vários blocos temporais determinados automaticamente pelo sistema oferecem ao utilizador uma narrativa ao longo do tempo.
+No próximo passo o sistema recorre ao algoritmo do *Conta-me Histórias* para criar um resumo das notícias mais importantes a partir do conjunto de documentos obtidos no Arquivo.pt. Cada bloco temporal determinado como relevante pelo sistema reúne um total de 20 notícias. Os vários blocos temporais determinados automaticamente pelo sistema oferecem ao usuário uma narrativa ao longo do tempo.
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -436,34 +415,35 @@ for period in summ_result["results"]:
 # Conclusões
 
-A web é hoje considerada uma ferramenta essencial de comunicação. Os arquivos web surgem neste contexto como um importante recurso de preservação dos conteúdos aí publicados. Embora o seu uso seja dominado por investigadores, historiadores ou jornalistas, o elevado volume de dados aí disponíveis sobre o nosso passado faz deste tipo de infra-estrutura uma fonte de recursos de elevado valor e extre ...
+A web é hoje considerada uma ferramenta essencial de comunicação. Neste contexto, os arquivos web surgem como um importante recurso de preservação dos conteúdos aí publicados. Embora o seu uso seja dominado por pesquisadores, historiadores ou jornalistas, o elevado volume de dados aí disponíveis sobre o nosso passado faz deste tipo de infraestrutura uma fonte de recursos de elevado valor e extrem ...
```
investigadores
utilizadores
utilizador
utilizadores

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -436,34 +415,35 @@ for period in summ_result["results"]:
+A web é hoje considerada uma ferramenta essencial de comunicação. Neste contexto, os arquivos web surgem como um importante recurso de preservação dos conteúdos aí publicados. Embora o seu uso seja dominado por pesquisadores, historiadores ou jornalistas, o elevado volume de dados aí disponíveis sobre o nosso passado faz deste tipo de infraestrutura uma fonte de recursos de elevado valor e extrem ...
 
-Neste tutorial, procurámos mostrar como criar automaticamente sumários temporais a partir de eventos coletados no passado fazendo uso dos dados obtidos do Arquivo.pt e da aplicação da biblioteca de sumarização temporal Conta-me Histórias. O tutorial aqui apresentado é um primeiro passo na tentativa de mostrarmos aos interessados na temática a forma simples como qualquer utilizador pode com concei ...
+Neste tutorial, procurámos mostrar como criar automaticamente sumários temporais a partir de eventos coletados no passado, fazendo uso dos dados obtidos no Arquivo.pt e da aplicação da biblioteca de sumarização temporal *Conta-me Histórias*. O tutorial aqui apresentado é um primeiro passo na tentativa de mostrarmos aos interessados na temática uma forma simples de como qualquer usuário pode, util ...
```
utilizador

joanacvp replied:
Respondido acima. Seguiram-se as diretrizes da equipa portuguesa do PH no que diz respeito ao termo.

rncampos commented on pt/esbocos/originais/sumarizacao-narrativas-web-python.md:
```diff
@@ -436,34 +415,35 @@ for period in summ_result["results"]:
 # Financiamento
 
-Ricardo Campos foi financiado pela [ERDF - European Regional Development Fund](https://ec.europa.eu/regional_policy/en/funding/erdf/) através do [Programa Operacional Regional do Norte (NORTE 2020)](https://www.norte2020.pt/), sob o programa Portugal 2020 e fundos nacionais da agência de financiamento à investigacao Portuguesa [Fundação para a Ciência e Tecnologia (FCT)](https://www.fct.pt/) com  ...
+Ricardo Campos foi financiado pelo [ERDF - European Regional Development Fund](https://ec.europa.eu/regional_policy/en/funding/erdf/) (em inglês) através do [Programa Operacional Regional do Norte (NORTE 2020)](https://www.norte2020.pt/), sob o programa Portugal 2020 e fundos nacionais da agência de financiamento da investigação portuguesa, [Fundação para a Ciência e Tecnologia (FCT)](https://www ...
```
Pf alterar pelo seguinte texto: Ricardo Campos foi financiado pelo [ERDF - European Regional Development Fund](https://ec.europa.eu/regional_policy/en/funding/erdf/) (em inglês) através do [Programa Operacional Regional do Norte (NORTE 2020)](https://www.norte2020.pt/) sob o Acordo de Parceria Portugal 2020 e por fundos nacionais da agência de financiamento da investigação portuguesa [Fundação para a Ciência e Tecnologia (FCT)](https://www.fct.pt/) com o projeto PTDC/CCI-COM/31857/2017 (NORTE-01-0145-FEDER-031857). Este financiamento faz parte da linha de pesquisa do projeto 'Text2Story.'

joanacvp replied:
Alterado. De acordo com as diretrizes modifiquei apenas a palavra "invesitgação" para "pesquisa".

joanacvp committed:
Update sumarizacao-narrativas-web-python.md

Alterações introduzidas mediante as revisões ao copyedit de @josircg e @rncampos

DanielAlvesLABDH reviewed (approved):
@joanacvp e @josircg as alterações feitas podem ser integradas? Da revisão que fiz, parece-me tudo bem.

joanacvp commented:
Da minha parte sim @DanielAlvesLABDH , podemos avançar. Posso fazer merge à pull request?

DanielAlvesLABDH commented:
Sim

joanacvp commented:
> Sim

Feito :)
